src/components/CreationForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useContext } from 'react'
+import { Link } from 'react-router-dom'
+import { GameContext } from '../store/GameContext'
+
+const initialState = {
+	code: '',
+	talent: '',
+	terms: '',
+	count: '',
+	intervals: ''
+}
+
+export const CreationForm = () => {
+	const { dispatch, state } = useContext(GameContext)
+
+	const handleChange = (event) => {
+		const { value, id } = event.target
+		initialState[id] = value
+		dispatch({ type: 'SET_CREATION_CARD', creationCard: initialState })
+	}
+
+	return (
+		<form className='creation-form'>
+			<label className='choice' htmlFor='talent'>
+				Choose a talent
+			</label>
+			<select className='talents' id='talent' value={state.talent} onChange={handleChange}>
+				<option hidden> Talents </option>
```

Comment:
remove whitespace, use CSS if needed

Reply:
What do you mean by white space?

Reply:
I might have been wrong about this being a bigger deal: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace

Reply:
I want to say it's the way github shares it files. You can maybe press `.` on the repo to explore it more accurately in there. I've also heard of plugins and extensions that make github reviews better (I haven't don't them enough to warrant the research). 

I am not sure how react handles white space but I don't care for them. even "ALL CAPS" I would rather just edit in the CSS using `text-transform: uppercase"  thanks for clarifying tho!